From b2ccbe3763316b39ec20c67206d4a9eccfd97092 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@nxp.com>
Date: Wed, 4 May 2016 14:55:11 +0800
Subject: [PATCH 1044/5242] MLK-12757-1 ARM: dts: imx6ull-ddr3-arm2.dts:
 change usdhc2 pad setting

commit  2d8f4a875c382aa1ce6a39290e0bc6f445f4a7a2 from
https://source.codeaurora.org/external/imx/linux-imx.git

According to Hardware team's suggestion, for usdhc2, this patch change
the drive strength for clock pin and data pin, which can make the signal
meet the requirement for DDR50 mode.

Signed-off-by: Haibo Chen <haibo.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
index 3642016..dd29e61 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2.dts
@@ -839,11 +839,11 @@
 		pinctrl_usdhc2_100mhz: usdhc2grp100mhz {
 			fsl,pins = <
 				MX6UL_PAD_NAND_WE_B__USDHC2_CMD	0x170b9
-				MX6UL_PAD_NAND_RE_B__USDHC2_CLK	0x100b9
-				MX6UL_PAD_NAND_DATA00__USDHC2_DATA0	0x170b9
-				MX6UL_PAD_NAND_DATA01__USDHC2_DATA1	0x170b9
-				MX6UL_PAD_NAND_DATA02__USDHC2_DATA2	0x170b9
-				MX6UL_PAD_NAND_DATA03__USDHC2_DATA3	0x170b9
+				MX6UL_PAD_NAND_RE_B__USDHC2_CLK	0x100a9
+				MX6UL_PAD_NAND_DATA00__USDHC2_DATA0	0x170a9
+				MX6UL_PAD_NAND_DATA01__USDHC2_DATA1	0x170a9
+				MX6UL_PAD_NAND_DATA02__USDHC2_DATA2	0x170a9
+				MX6UL_PAD_NAND_DATA03__USDHC2_DATA3	0x170a9
 			>;
 		};
 
-- 
1.7.9.5

